### ISSUE TITLE
[SF][Tools]: Error thrown when running `build.jsImports.illustrations`

### ISSUE THREAD

**Issue by zhangliangyi** [SAP/ui5-webcomponents#5992]
### **Bug Description**
Error thrown when running `build.jsImports.illustrations` task with `@ui5/webcomponents-tools@1.9.0`

### **Expected Behavior**
I suppose the line https://github.com/SAP/ui5-webcomponents/blob/93ef1315a2ce9ca49b594e5de5b15f4709900e54/packages/tools/components-package/nps.js#L4
should be
```javascript
const FIORI = path.join(__dirname, `../../webcomponents-fiori/`);
```

I tried to find the package named `@ui5/fiori` in npm, but no result returned. So I believe the path above is not correct.

### **Context**
- UI5 Web Components version: 1.9.0
- OS/Platform: MacOS
- Browser: N/A
- Affected component: tools

### **Log Output / Stack Trace / Screenshots**
```
[2] [Error: ENOENT: no such file or directory, scandir '/Users/i307840/Workspace/xweb/common-webcomponents/node_modules/@ui5/fiori/dist/illustrations'] {
[2]   errno: -2,
[2]   code: 'ENOENT',
[2]   syscall: 'scandir',
[2]   path: '/Users/i307840/Workspace/xweb/common-webcomponents/node_modules/@ui5/fiori/dist/illustrations'
[2] }
[2] The script called "build.jsImports.illustrations" which runs "node "/Users/i307840/Workspace/xweb/common-webcomponents/node_modules/@ui5/webcomponents-tools/lib//generate-js-imports/illustrations.js" /Users/i307840/Workspace/xweb/common-webcomponents/node_modules/@ui5/fiori//dist/illustrations /Users/i307840/Workspace/xweb/common-webcomponents/node_modules/@ui5/fiori//dist/illustrations/tnt dist/generated/js-imports" failed with exit code 1 https://github.com/sezna/nps/blob/master/other/ERRORS_AND_WARNINGS.md#failed-with-exit-code
[2] The script called "build.jsImports" which runs "mkdirp dist/generated/js-imports && nps build.jsImports.illustrations" failed with exit code 1 https://github.com/sezna/nps/blob/master/other/ERRORS_AND_WARNINGS.md#failed-with-exit-code
[2] The script called "prepare.styleRelated" which runs "nps build.styles build.jsonImports build.jsImports" failed with exit code 1 https://github.com/sezna/nps/blob/master/other/ERRORS_AND_WARNINGS.md#failed-with-exit-code
[2] nps prepare.styleRelated exited with code 1
The script called "prepare.all" which runs "concurrently "nps build.templates" "nps build.i18n" "nps prepare.styleRelated" "nps copy" "nps build.api" "nps build.illustrations"" failed with exit code 1 https://github.com/sezna/nps/blob/master/other/ERRORS_AND_WARNINGS.md#failed-with-exit-code
The script called "prepare" which runs "nps clean prepare.all" failed with exit code 1 https://github.com/sezna/nps/blob/master/other/ERRORS_AND_WARNINGS.md#failed-with-exit-code
The script called "build" which runs "nps lint prepare build.bundle" failed with exit code 1 https://github.com/sezna/nps/blob/master/other/ERRORS_AND_WARNINGS.md#failed-with-exit-code
node:child_process:915
    throw err;
    ^

Error: Command failed: npx nps "build"
    at checkExecSyncError (node:child_process:841:11)
    at Object.execSync (node:child_process:912:15)
    at Object.<anonymous> (/Users/i307840/Workspace/xweb/common-webcomponents/node_modules/@ui5/webcomponents-tools/bin/dev.js:16:15)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 81915,
  stdout: null,
  stderr: null
}
```
### **Priority**
- [ ] Low
- [ ] Medium
- [ ] High
- [x] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SF
- Business impact: {...}

**Comment by ilhan007** [comment]
Hello @zhangliangyi 
the fix has been released with 1.9.1. Please give it a try and let us know how it works.

BR,
ilhan

**Comment by zhangliangyi** [comment]
@ilhan007 
It works for us with the latest 1.9.1, thanks for the quick fix.
